core/arch/arm/plat-sam/scmi_server.c

```diff
@@ -74,6 +75,133 @@ const uint8_t *plat_scmi_protocol_list(unsigned int channel_id __unused)
 	return plat_protocol_list;
 }
 
+struct sama5d2_pmc_clk {
+	unsigned int scmi_id;
+	unsigned int pmc_type;
+	unsigned int pmc_id;
+};
+
+static const struct sama5d2_pmc_clk pmc_clks[] = {
+	{ AT91_SCMI_CLK_CORE_MCK, PMC_TYPE_CORE, PMC_MCK },
```

Comment:
nitpicking: would be better with named fields.

Reply:
Done